src/utils/stackingUtils.ts

```diff
@@ -11,6 +11,8 @@ import { IAccessor } from "../core/interfaces";
 import * as Utils from "./";
 import { makeEnum } from "./makeEnum";
 
+const lMemoize = require("lodash.memoize");
```

Comment:
even though this keeps the library size smaller, i think i'd prefer to use the normal import syntax. Unfortunately the modular lodash packages don't support `import` statements for some reason.

Reply:
There are some instructions on how to get treeshaking to work with lodash, but i couldn't get it to work

https://medium.com/@martin_hotell/tree-shake-lodash-with-webpack-jest-and-typescript-2734fa13b5cd